### ISSUE TITLE
Screen readers do not announce changes that automatically occur upon entering Email in the edit field, leaving users unaware of dynamically updated information.

### ISSUE THREAD

**Issue by msftedad** [adobe/adobe.github.com#150]
### Expected Behaviour
 Screen Readers should announce automatic changes in edit fields to keep users informed of dynamically updated information, like loading or progress info and once the page has loaded. It should announce the page title as well.
### Actual Behaviour
Users do not receive auditory feedback from screen readers when automatic changes occur on entering Email ID in the edit field. Screen reader remains silent once the new page opens.

https://github.com/user-attachments/assets/565b7e98-4c3b-43b8-a9e7-9faaf93fd37d

#### Steps to Reproduce

1. Open URL [https://www.adobe.com/acrobat/pdf-reader.html](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fwww.adobe.com%2facrobat%2fpdf-reader.html&data=05%7c02%7cv-ymounika%40microsoft.com%7cfc695922b7c34646c2c008dcbb5ebd70%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c638591262795256923%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c0%7c%7c%7c&sdata=o9PY7goIjRXOoVaVs7vxiVonXaIBKfShZ6rnvhgxgDw%3D&reserved=0) in edge browser.
2. Turn on Screen reader using 'Ctrl+ Win+ Enter'.
3. Navigate to the Sing button and activate it.
4. Sign in page will be open.
5. Choose 'Continue with Google’ to sign-in.
6. Enter Email ID and activate Continue button. Page starts to load and redirects to Welcome back page.
7. Screen Reader will announce dynamically updated information.
8. Logout from already login account.
9. Again, try to login with same credentials.
10. Verify screen readers announcement, whether it is announcing the “Welcome back page” information or not.

#### Platform and Version
OS: Windows 11 Version 24H2 (OS Build 26100.1301)
Browser: Microsoft New Edge Version 127.0.2651.86 (Official build) (64-bit)

Test Credentials: testu4496@gmail.com / !India@123
#### Sample Code that illustrates the problem
[https://www.adobe.com/acrobat/pdf-reader.html](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fwww.adobe.com%2facrobat%2fpdf-reader.html&data=05%7c02%7cv-ymounika%40microsoft.com%7cfc695922b7c34646c2c008dcbb5ebd70%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c638591262795256923%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c0%7c%7c%7c&sdata=o9PY7goIjRXOoVaVs7vxiVonXaIBKfShZ6rnvhgxgDw%3D&reserved=0)
#### Logs taken while reproducing problem
<img width="705" alt="Screen readers do not announce changes that automatically occur upon entering Email in the edit field" src="https://github.com/user-attachments/assets/217b4d4a-9fd0-40e6-a796-ea7ef13d7216">

**Comment by msftedad** [comment]
Hi Team, Any update on this?

**Comment by pragya411** [comment]
Hi Team, Any update?

**Comment by msftedad** [comment]
Hi Team, Any update?

**Comment by msftedad** [comment]
Hi Team, 
This issue is still repro'ing. Please Provide an update on this issue.
